src/test/java/com/emc/object/s3/S3JerseyClientTest.java

```diff
@@ -2278,6 +2275,23 @@ public void testPreSignedUrlHeaderOverrides() throws Exception {
         Assert.assertEquals(contentDisposition, response.getHeaders().getFirst(RestUtil.HEADER_CONTENT_DISPOSITION));
     }
 
+    @Test
```

Comment:
add a comment here to say you must pause, and manually verify the vpool on the bucket in the GUI... unfortunately there is no way to look at the vpool of a bucket via S3.

Reply:
✔️